index.html

```diff
@@ -14,46 +14,258 @@
+
+      <!-- stories section -->
+      <section class="stories-section">
+        <div class="storysection-image"></div>
+        <h2 class="multi-Post">
+          Multi-Post<br />
+          Stories
+        </h2>
+        <div class="canopy-container">
+          <h2 class="canopy">CANOPY</h2>
+          <p class="date">• Back End Dev • 2015</p>
+        </div>
+        <p class="daily-sign">
+          A daily selection of privately<br />
+          personalized reads; no accounts or sign-ups required.
+        </p>
+
+        <div class="button-container">
+          <button class="html">html</button>
+          <button class="css">css</button>
+          <button class="javascript">javaScript</button>
+        </div>
+
+        <button class="btn">See Project</button>
+      </section>
+
+      <section class="about-section">
+        <h2 class="myself">
+          About<br />
+          Myself
+        </h2>
+        <p class="career">
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <h4 class="connect">LET’S CONNECT</h4>
+
+        <ul class="social-icons">
+          <li><span class="iconify" data-icon="iconoir:github"></span></li>
+          <li><span class="iconify" data-icon="eva:linkedin-fill"></span></li>
+          <li><span class="iconify" data-icon="fontisto:angelist"></span></li>
+          <li><span class="iconify" data-icon="ei:sc-twitter"></span></li>
+          <li><span class="iconify" data-icon="bx:bxl-medium"></span></li>
+        </ul>
+
+        <button class="resume">Get my resume</button>
+
+        <ul class="language-list">
+          <li class="langguage">
+            <h5 class="icondown-language">Langguages</h5>
+            <div class="icon-down"></div>
+          </li>
+
+          <li class="major-lang">
+            <img src="img/Ellipse 1.png" class="eclipse-language" />
+            <p class="technology-language">JavaScript</p>
+          </li>
+          <li class="major-lang">
+            <img src="img/Ellipse 1 (1).png" class="eclipse-language" />
+            <p class="technology-language">HTML</p>
+          </li>
+          <li class="major-lang">
+            <img src="img/Ellipse 1 (2).png" class="eclipse-language" />
+            <p class="technology-language">CSS</p>
+          </li>
+          <li class="major-lang">
+            <h5 class="about-h5">Frameworks</h5>
+            <div class="icon-right"></div>
+          </li>
+
+          <li class="major-lang">
+            <h5 class="about-h5">Skills</h5>
+            <div class="icon-right"></div>
+          </li>
+        </ul>
+      </section>
+
+      <section class="contact-section">
+        <div class="ellipse-rectangle">
+          <img src="img/Ellipse 12.png" alt="Eclipse-image" class="Ellipse12" />
```

Comment:
- [ ] Decoration images should not be part of the HTML. They should be in the `contact-section class inside the CSS.
please remove them from here and set them as a background image.
You can download the background from the figma page let me show you.
 ## Download the background
![how-to-download](https://user-images.githubusercontent.com/28903404/143721503-e52ca172-df84-4e2b-abc5-792ca1aa62e5.gif)

After getting the background you can position it using `background-position` This way the background images will automatically adapt to different screen changes and they won't mess with other elements in the HTML.